generative_ai/prompts/prompt_optimizer.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+
+#  [START generativeaionvertexai_prompt_optimizer]
```

Comment:
fix: According to latest recommendation (go/runnable-snippets) the opening tag should start inside the function to make the sample executable as-is (without a need to call the function)

Reply:
done!